client/app/scripts/superdesk-authoring/views/authoring-topbar.html

```diff
@@ -87,9 +78,22 @@
                     </li>
                 </ul>
 
-                <ul ng-if="_editable"><li class="divider"></li></ul>
+                <ul ng-if="item.task && item.task.desk && itemActions.mark_item"
```

Comment:
You probably mean `item.task.desk && itemActions.mark_item` ?
Yes, it is, Angular gracefully handles the cases when `item.task` is undefined.